gen/definitions/snmp_server_view.yaml

```diff
@@ -0,0 +1,18 @@
+name: SNMP Server View
+path: Cisco-IOS-XR-um-snmp-server-cfg:snmp-server/views/view[view-name-%s]
+doc_category: Management
+attributes:
+  - yang_name: view-name
+    example: VIEW12
+
+  - yang_name: mib-view-families/mib-view-family
+    tf_name: mib_view_families
+    type: List
+    attributes:
+      - yang_name: mib-view-family-name
```

Comment:
Can we add a tf_name here as `mib_view_family` is redundant:
```
tf_name: name
```

Reply:
The list element name was fine, what I meant wan renaming the `mib-view-family-name` attribute to `name`.

```
  - yang_name: mib-view-families/mib-view-family
    tf_name: mib_view_families
    type: List
    attributes:
      - yang_name: mib-view-family-name
        tf_name: name
```

Reply:
Please change back the `tf_name` of the list like in my snippet above.